mingw-w64-arrow/PKGBUILD

```
@@ -48,10 +48,10 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
              "${MINGW_PACKAGE_PREFIX}-xsimd")
 source=("https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-${pkgver}/apache-arrow-${pkgver}.tar.gz"
         "https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-${pkgver}/apache-arrow-${pkgver}.tar.gz.asc"
-        "001-support-llvm-18.patch::https://github.com/apache/arrow/commit/ed78986a.patch")
-sha256sums=('abbf97176db6a9e8186fe005e93320dac27c64562755c77de50a882eb6179ac6'
+        "0001-find-protobuf-config-instead-of-module.patch")
```

Could you use the merged change instead? (We also need to update `sha256sum`.)

```suggestion
        "0001-find-protobuf-config-instead-of-module.patch::https://github.com/apache/arrow/commit/64be7a2.patch")
```

---

I've updated the patch and the sha256 but I don't seem to know how to calculate the correct sha for a patch download like this. I've tried with this:
```
$ curl https://github.com/apache/arrow/commit/64be7a2.patch -o 0001-find-protobuf-config-instead-of-module.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2511  100  2511    0     0  13947      0 --:--:-- --:--:-- --:--:-- 14027
$ sha256sum 0001-find-protobuf-config-instead-of-module.patch 
2eb48799459ea196bf66e0578a97bd7f0ccc1072f6b48c8b260f80f8740ec11f  0001-find-protobuf-config-instead-of-module.patch
```
but this doesn't seem to be the correct sha. Can someone help me understand how I can check the sha for this?